packages/appium/lib/logsink.js

```diff
@@ -186,8 +186,26 @@ async function createTransports(args) {
   return transports;
 }
 
+let encounteredPrefixes = [];
+
+function getColorizedPrefix(prefix) {
+  let prefixId = prefix.split('@')[0].trim();
+  prefixId = prefixId.split(' (')[0].trim();
+  if (encounteredPrefixes.indexOf(prefixId) < 0) {
+    encounteredPrefixes.push(prefixId);
+  }
+  // using a multiple of 16 should cause 16 colors to be created
+  const colorNumber = encounteredPrefixes.indexOf(prefixId) * 16;
```

Comment:
so once we pass 16 prefixes, we stop colourizing? what about just cycling around? i think i'd rather see two prefixes share a colour than certain prefixes not have a colour?

Reply:
Great idea! We can modulus 256, and always have a good color. Will do this!

Reply:
This has been resolved